test/test_ssl.rb

```diff
@@ -865,6 +865,32 @@ def test_servername_cb_raises_an_exception_on_unknown_objects
     sock2.close if sock2
   end
 
+  def test_accept_errors_include_peeraddr
+    context = OpenSSL::SSL::SSLContext.new
+    context.cert = @svr_cert
+    context.key = @svr_key
+
+    server = TCPServer.new("127.0.0.1", 0)
+    port = server.connect_address.ip_port
+
+    ssl_server = OpenSSL::SSL::SSLServer.new(server, context)
+
+    t = Thread.new do
+      assert_raise_with_message(OpenSSL::SSL::SSLError, /peeraddr=127\.0\.0\.1/) do
```

Comment:
Maybe another way to do this, rather than adjusting the log message, is to add `SSLError#peeraddr` which is sufficient for the initial issue. What do you think?

Reply:
I think it makes sense. I will have to dig into how `SSLError` is implemented to include the peeraddr.

Reply:
Not all SSLErrors need a peeraddr, so could make a subclass, e.g. `SSLConnectionError`.